src/olympia/users/tests/test_models.py

```diff
@@ -795,26 +795,83 @@ def test_blocked_user_login_ip(self):
         request = RequestFactory(REMOTE_ADDR='192.168.0.8').get('/')
         request.user = user_factory(last_login_ip='192.168.1.1')
         IPNetworkUserRestriction.objects.create(network='192.168.1.1/32')
-        assert not IPNetworkUserRestriction.allow_request(request)
+        assert not IPNetworkUserRestriction.allow_submission(request)
+
+    def test_allowed_approval_while_blocking_submission(self):
+        request = RequestFactory(REMOTE_ADDR='192.168.0.1').get('/')
+        request.user = user_factory(last_login_ip='10.0.0.1')
+        IPNetworkUserRestriction.objects.create(network='192.168.0.0/28')
+        # Submission is allowed.
+        assert not IPNetworkUserRestriction.allow_submission(request)
```

Comment:
The comments say `# Submission is allowed.` and `# Approval is blocked.`, but the assertions seems to be the opposite, and the name of the test also indicates that we expect the opposite. I am guessing the comments are wrong?

Reply:
Yes, I'll fix